app/Http/Controllers/SubscriptionController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Newsletter;
+
+class SubscriptionController extends Controller
+{
+    public function subscribe(Request $request)
```

Comment:
Nit pick, for a more standard route name:
```suggestion
    public function store(Request $request)
```